Framework/PythonInterface/test/python/plugins/algorithms/WorkflowAlgorithms/IndirectTwoPeakFitTest.py

```diff
@@ -0,0 +1,88 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+from __future__ import (absolute_import, division, print_function)
+
+from mantid.api import (AnalysisDataService, MatrixWorkspace, WorkspaceGroup)
+from mantid.simpleapi import (CompareWorkspaces, EnergyWindowScan, IndirectTwoPeakFit, LoadNexus)
+
+import unittest
+
+
+def exists_in_ads(workspace_name):
+    return AnalysisDataService.doesExist(workspace_name)
+
+
+def get_ads_workspace(workspace_name):
+    return AnalysisDataService.retrieve(workspace_name) if exists_in_ads(workspace_name) else None
+
+
+class IndirectTwoPeakFitTest(unittest.TestCase):
+
+    def setUp(self):
+        EnergyWindowScan(InputFiles='OSIRIS92762', Instrument='OSIRIS', Analyser='graphite', Reflection='002',
+                         SpectraRange='963,980', ElasticRange='-0.02,0.02', InelasticRange='0.4,0.5',
+                         TotalRange='-0.5,0.5', ReducedWorkspace='__reduced_group', ScanWorkspace='__scan_workspace')
+
+        self._red_name = 'osiris92762_graphite002_red'
+        self._output_name = 'osiris92762_graphite002_two_peak_fit'
+
+        self._x_min = -0.927689707342884
+        self._x_max = 2.1502418039357183
```

Comment:
does it really have to be such high precision?

Reply:
@AnthonyLim23 I think -0.5, 0.5 should be ok